attributes/default.rb

```diff
@@ -23,6 +23,12 @@
 #
 default['chef-ingredient']['mixlib-install']['git_ref'] = nil
 
+#
+# Optionally override the omnitruck endpoint
+#
+default['chef-ingredient']['omnitruck']['endoint'] = nil
```

Comment:
I thought we were moving to a place where Omnitruck is not in the mix? We should be able to install all products using the mixlib-install's Bintray backend.